client/src/featureform/cli.py

```diff
@@ -148,6 +149,11 @@ def list(host, cert, insecure, local, resource_type):
 app.register_blueprint(dashboard_app)
 
 
+@cli.command()
+def version():
+    print("Client Version: {}".format(get_package_version()))
```

Comment:
Would we be able to do a GET here and if FEATUREFORM_HOST env var is set then get the cluster version?
Then have:
```
Client Version:
Cluster Version:
```
```
If FEATUREFORM_HOST != ""
try:
    version = Get <host>/data/version
except:
    version = "Could not fetch version"

```

Reply:
Yeah that should be possible, I'll give it a shot and report back.

Reply:
and done. I'll get with you first to double check some assumptions. 

<img width="809" alt="Screenshot 2023-05-26 at 2 48 27 PM" src="https://github.com/featureform/featureform/assets/34227093/329b0174-e051-43aa-b0af-c64b9998866b">

Reply:
and done. I'll get with you first to double check some assumptions.
<img width="620" alt="image" src="https://github.com/featureform/featureform/assets/34227093/c7660adf-7606-4334-b056-f78cd454e02e">